x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/aggregate/Count.java

```diff
@@ -94,11 +98,17 @@ public Count(
     }
 
     public Count(Source source, Expression field, Expression filter) {
+        this(source, field, filter, false);
+    }
+
+    private Count(Source source, Expression field, Expression filter, boolean isSampleCorrected) {
         super(source, field, filter, emptyList());
+        this.isSampleCorrected = isSampleCorrected;
     }
 
     private Count(StreamInput in) throws IOException {
         super(in);
+        this.isSampleCorrected = false;
```

Comment:
Let's add a comment here too, why this should always be `false` on the deserializing end.